### ISSUE TITLE
Cannot add page(s)

### ISSUE THREAD

**Issue by idasbv** [dnnsoftware/Dnn.Platform#3447]
## Description of bug
Cannot add pages and page tree is not visible

## Steps to reproduce
List the precise steps to reproduce the bug:
1. Go to 'Pages'
2. Click on 'Add Page'

## Current behavior
Error: 'There was een error retrieving your content'

## Expected behavior
Add page interface to add pagename(s)

## Screenshots
![image](https://user-images.githubusercontent.com/13104302/71712746-ec751900-2e06-11ea-9307-89d5f436054a.png)

## Error information
Provide any error information (console errors, error logs, etc.) related to this bug.

## Affected version

* [ ] 9.4.4 clean install
* [ ] 9.4.4 upgrade

## Affected browser
All

**Comment by valadas** [comment]
I cannot reproduce, do you have any logs in either:

1. The browser console
2. The admin logs
3. In the /portals/_default/logs folder

**Comment by idasbv** [comment]
log is empty...

**Comment by idasbv** [comment]
We added some pages yesterday without problems... but today we get this error...

**Comment by idasbv** [comment]
console error: <Error>
<Message>An error has occurred.</Message>
<ExceptionMessage>
An error occurred when trying to create a controller of type 'PagesController'. Make sure that the controller has a parameterless public constructor.
</ExceptionMessage>
<ExceptionType>System.InvalidOperationException</ExceptionType>
<StackTrace>
at System.Web.Http.Dispatcher.DefaultHttpControllerActivator.Create(HttpRequestMessage request, HttpControllerDescriptor controllerDescriptor, Type controllerType) at System.Web.Http.Tracing.Tracers.HttpControllerActivatorTracer.<>c__DisplayClass2.<System.Web.Http.Dispatcher.IHttpControllerActivator.Create>b__0() at System.Web.Http.Tracing.ITraceWriterExtensions.TraceBeginEnd(ITraceWriter traceWriter, HttpRequestMessage request, String category, TraceLevel level, String operatorName, String operationName, Action`1 beginTrace, Action execute, Action`1 endTrace, Action`1 errorTrace) at System.Web.Http.Tracing.Tracers.HttpControllerActivatorTracer.System.Web.Http.Dispatcher.IHttpControllerActivator.Create(HttpRequestMessage request, HttpControllerDescriptor controllerDescriptor, Type controllerType) at System.Web.Http.Controllers.HttpControllerDescriptor.CreateController(HttpRequestMessage request) at System.Web.Http.Tracing.Tracers.HttpControllerDescriptorTracer.<>c__DisplayClass2.<CreateController>b__0() at System.Web.Http.Tracing.ITraceWriterExtensions.TraceBeginEnd(ITraceWriter traceWriter, HttpRequestMessage request, String category, TraceLevel level, String operatorName, String operationName, Action`1 beginTrace, Action execute, Action`1 endTrace, Action`1 errorTrace) at System.Web.Http.Tracing.Tracers.HttpControllerDescriptorTracer.CreateController(HttpRequestMessage request) at System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__1.MoveNext()
</StackTrace>
<InnerException>
<Message>An error has occurred.</Message>
<ExceptionMessage>
Unable to resolve DotNetNuke.Abstractions.INavigationManager as parameter "navigationManager" in scoped Dnn.PersonaBar.Pages.Services.PagesController. Where no service registrations found and number of Rules.FallbackContainers: 1 and number of Rules.UnknownServiceResolvers: 0
</ExceptionMessage>
<ExceptionType>DnnSharp.DryIoc.ContainerException</ExceptionType>
<StackTrace>
at DnnSharp.DryIoc.Throw.It(Int32 error, Object arg0, Object arg1, Object arg2, Object arg3) at DnnSharp.DryIoc.Container.ThrowUnableToResolve(Request request) at DnnSharp.DryIoc.Container.DnnSharp.DryIoc.IContainer.ResolveFactory(Request request) at DnnSharp.DryIoc.ReflectionFactory.CreateExpressionOrDefault(Request request) at DnnSharp.DryIoc.Factory.GetExpressionOrDefault(Request request) at DnnSharp.DryIoc.Factory.GetDelegateOrDefault(Request request) at DnnSharp.DryIoc.Container.ResolveAndCacheDefaultDelegate(Type serviceType, Boolean ifUnresolvedReturnDefault) at DnnSharp.DryIoc.Container.DnnSharp.DryIoc.IResolver.Resolve(Type serviceType, Boolean ifUnresolvedReturnDefault) at System.Web.Http.Dispatcher.DefaultHttpControllerActivator.GetInstanceOrActivator(HttpRequestMessage request, Type controllerType, Func`1& activator) at System.Web.Http.Dispatcher.DefaultHttpControllerActivator.Create(HttpRequestMessage request, HttpControllerDescriptor controllerDescriptor, Type controllerType)
</StackTrace>
</InnerException>
</Error>

**Comment by idasbv** [comment]
Maybe an issue with a DNN Sharp module?

**Comment by idasbv** [comment]
Actionform is the problem... we will upgrade to the latest version first (Hotfix)

**Comment by david-poindexter** [comment]
@idasbv glad you figured it out and thanks for letting us know.  We will go ahead and close this issue.

**Comment by idasbv** [comment]
We found out the hotfix also doesn't work... The modules simple isn't compatible with DNN 9.4 because of the changes in some dependency injections... This is quite an issues for all people that use the DNNSharp modules...

**Comment by valadas** [comment]
Did you used 9.4.4? Did you get the latest DnnSharp version of their modules? I remember there was an issue but I think it was resolved in latest releases...

**Comment by jdomian** [comment]
This is still an issue. We have security requirements for DNN sites that are less than 9.4.1 that also rely heavily on ActionForms version 05.00.155. I tried a "clean" install with no modules, only DNN 9.4.4 and I could create pages, and do everything as expected through the UI. Then installed ActionForms 05.00.155 and it has the same "broken" behavior as the site currently under development we are working on. After that, I saw DNNSharp had a HotFix for ActionForms v05.00.603, still same issues.
